_themes/citus/layout.html

```diff
@@ -164,9 +164,10 @@
       <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
         <a href="{{ pathto(master_doc) }}">{{ project }}</a>
       <ul>
-        <li><a href="/product/citus">Product</a></li>
-        <li><a href="/solutions/applications">Solutions</a></li>
-        <li><a href="/blog">Blog</a></li>
+        <li><a href="https://www.citusdata.com/product/citus">Product</a></li>
```

Comment:
After changing version numbers in a bunch of postgres docs links I ran a program checking for broken links in all these changes. Did not find any from the version renaming but did find broken links in the mobile nav menu. I Included the fix in this PR as well.